www/include/views/graphTemplates/DB-Func.php

```diff
@@ -49,50 +50,68 @@ function testExistence($name = null)
         htmlentities($name, ENT_QUOTES, "UTF-8") . "'";
     $res = $pearDB->query($query);
     $graph = $res->fetch();
-    /*
-	 * Modif case
-	 */
+    // Modif case
     if ($res->rowCount() >= 1 && $graph["graph_id"] == $id) {
         return true;
-    } /*
-	 * Duplicate entry
-	 */
-    elseif ($res->rowCount() >= 1 && $graph["graph_id"] != $id) {
+    } elseif ($res->rowCount() >= 1 && $graph["graph_id"] != $id) {
+        // duplicate entry
         return false;
     } else {
         return true;
     }
 }
 
```

Comment:
couldn't you can invert the elseif and if to avoid the else ?
if ($res->rowCount() >= 1 && $graph["graph_id"] != $id) {
return false;
} else {
return true;
}

Reply:
I could but I will not change this in this PR :-1: